packages/devextreme-react/src/core/__tests__/test-component.ts

```diff
@@ -36,6 +36,16 @@ class TestComponent<P = any> extends Component<P & IHtmlOptions> {
 
   protected useDeferUpdateFlag = true;
 
+  get element(): HTMLElement {
```

Comment:
Is this redundant since you now expose `element` via `instance.element()`?

Reply:
Yeah, probably, but i use it in instance getter 
Maybe it might be useful in other tests